ddl/ddl_tiflash_test.go

```diff
@@ -60,6 +60,8 @@ const (
 func (s *tiflashDDLTestSuite) SetUpSuite(c *C) {
```

Comment:
You need migrate those new test case to new test framework.

Reply:
ref #26022

Reply:
https://github.com/pingcap/tidb/pull/31946 will do that